web/src/app/admin/skills/shared/skills.serviceApp.ts

```diff
@@ -7,6 +7,9 @@ import { SkillService } from '../../../webapi/services/skill.service';
 @Injectable()
 export class SkillsServiceApp {
 
+  onBlurMethod():Observable<any>  {
```

Comment:
what this method is doing ? Change method name

Reply:
done